datasets/ccaligned_multilingual/ccaligned_multilingual.py

```diff
@@ -0,0 +1,216 @@
+            description="The dataset containing sentence-pairs for en_XX-zz_TR.",
+        ),
+        CcalignedMultilingualConfig(
+            type="documents",
+            language_code="tz_MA",
+            version=VERSION,
+            description="The dataset containing document-pairs for en_XX-tz_MA.",
+        ),
+        CcalignedMultilingualConfig(
+            type="sentences",
+            language_code="tz_MA",
+            version=VERSION,
+            description="The dataset containing sentence-pairs for en_XX-tz_MA.",
+        ),
+        CcalignedMultilingualConfig(
+            type="documents",
+            language_code="ak_GH",
+            version=VERSION,
+            description="The dataset containing document-pairs for en_XX-ak_GH.",
+        ),
+        CcalignedMultilingualConfig(
+            type="sentences",
+            language_code="ak_GH",
+            version=VERSION,
+            description="The dataset containing sentence-pairs for en_XX-ak_GH.",
+        ),
+    ]
+
+    BUILDER_CONFIG_CLASS = CcalignedMultilingualConfig
+
+    # DEFAULT_CONFIG_NAME = "documents-zz_TR" # Not Needed
+
+    def _info(self):
+        if self.config.name[:9] == "documents":
+            features = datasets.Features(
+                {
+                    "Domain": datasets.Value("string"),
+                    "Source_URL": datasets.Value("string"),
+                    "Target_URL": datasets.Value("string"),
+                    "translation": datasets.Translation(languages=("en_XX", self.config.language_code)),
+                }
+            )
+        else:
+            features = datasets.Features(
+                {
+                    "translation": datasets.Translation(languages=("en_XX", self.config.language_code)),
+                    "LASER_similarity": datasets.Value("float"),
+                }
+            )
+
+        return datasets.DatasetInfo(
+            # This is the description that will appear on the datasets page.
+            description=_DESCRIPTION,
+            # This defines the different columns of the dataset and their types
+            features=features,  # Here we define them above because they are different between the two configurations
+            supervised_keys=None,
+            # Homepage of the dataset for documentation
+            homepage=_HOMEPAGE,
+            # License for the dataset if available
+            license=_LICENSE,
+            # Citation for the dataset
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        """Returns SplitGenerators."""
+        my_urls = _URLs[self.config.name[:9]]
+        if self.config.name[:9] == "sentences" and self.config.language_code in reverse_mapped_sentences:
+            url = os.path.join(my_urls, self.config.language_code + "-en_XX.tsv.xz")
+            from_english = False
+        else:
+            url = os.path.join(my_urls, "en_XX-" + self.config.language_code + ".tsv.xz")
```

Comment:
```suggestion
            url = my_urls + self.config.language_code + "-en_XX.tsv.xz"
            from_english = False
        else:
            url = my_urls + "en_XX-" + self.config.language_code + ".tsv.xz"
```
don't use os.path.join on urls since the separator is a backslash on windows, and not a slash.